Packs/Core/Playbooks/playbook-Exchange_forwarding_rule_configured.yml

```diff
@@ -0,0 +1,1736 @@
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -410,
+          "y": 2135
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: ec931801-d2f1-4042-84b4-0a7adf76ed05
+    type: title
+    task:
+      id: ec931801-d2f1-4042-84b4-0a7adf76ed05
+      version: -1
+      name: Hard Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "32"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 150,
+          "y": 1310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: f6057cef-9260-4970-8b1d-88edb25b4059
+    type: title
+    task:
+      id: f6057cef-9260-4970-8b1d-88edb25b4059
+      version: -1
+      name: Soft Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "15"
+      - "20"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": 1310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "38":
+    id: "38"
+    taskid: 3b4840c0-990a-42d1-835b-a3bc6e18f5ad
+    type: condition
+    task:
+      id: 3b4840c0-990a-42d1-835b-a3bc6e18f5ad
+      version: -1
+      name: Check user notification requirement
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      "yes":
+      - "39"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.SendNotification
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 940,
+          "y": 1930
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 49505509-b341-41dd-83e9-c80018d07814
+    type: regular
+    task:
+      id: 49505509-b341-41dd-83e9-c80018d07814
+      version: -1
+      name: Send the user a notification via email
+      description: Sends an email.
+      script: EWSO365|||send-mail
```

Comment:
I don't see a test in the playbook that checks if EWS is available. If the customer did not configure EWS, this task will fail.

Reply:
I used skip if unavailable.